src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -707,6 +702,9 @@ public static String getCopyrightAcceptUrl(String copyrightAlert,
 		// update folder name if there is any parent folder renaming
 		// checks for folder name updates in the path
 		// replace all old folder title with new title
+		if (title.contains("/")) {
+			title = title.replace('/', '_');
```

Comment:
This one replace all of them with underscore Like `the folder / this / name` with `the folder_this_name`